src/main/java/mate/academy/rickandmorty/service/PullDataServiceImpl.java

```diff
@@ -0,0 +1,60 @@
+package mate.academy.rickandmorty.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import jakarta.annotation.PostConstruct;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.config.mapper.CharacterMapper;
+import mate.academy.rickandmorty.dto.external.ApiResponseDto;
+import mate.academy.rickandmorty.dto.external.CharacterResponseDataDto;
+import mate.academy.rickandmorty.exception.ApiResponseParsingException;
+import mate.academy.rickandmorty.model.CharacterRickAndMorty;
+import mate.academy.rickandmorty.repository.CharacterRepository;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+@RequiredArgsConstructor
+@Service
+public class PullDataServiceImpl implements PullDataService {
+    @Value("https://rickandmortyapi.com/api/character")
```

Comment:
```suggestion
    @Value("${rick-and-morty.url}")
```

add `rick-and-morty.url=https://rickandmortyapi.com/api/character` to application.properties